src/haddock/clis/re/clustfcc.py

```diff
@@ -167,5 +183,36 @@ def reclustfcc(
         if caprieval_folder:
             log.info("Rewriting capri tables")
             rewrite_capri_tables(caprieval_folder, clt_dic, outdir)
+    
+    else:
+        output_models = models
```

Comment:
why this? are the `output_models` necessary in an interactive running?

Reply:
Mandatory, as the matrix plot is using `output_models` to get the current list of models